kubernetes/zenko/templates/cosmos-operator/clusterrole.yaml

```diff
@@ -0,0 +1,23 @@
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: {{ template "cosmos-operator.fullname" . }}
+  labels:
+    app: {{ template "cosmos-operator.name" . }}
+    chart: {{ template "zenko.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+rules:
+- apiGroups: [""]
+  resources:
+  - persistentvolumeclaims
+  - persistentvolumes
+  verbs:
+  - "*"
```

Comment:
I'm not sure but do we need all the verbs? For PVs don't we just need create/delete?

Reply:
We probably need `watch` and `list` verbs too. Perhaps `update` and `patch` are used internally by the opeartor for meta-stuff. I don't see a problem in giving  more permissions if we already need `create` and `delete` since those are the most "dangerous" ones.

Reply:
Yeah but the whole point is to only give access where explicitly needed. Just because `create` and `delete` are required doesn't mean it's okay to needlessly give out permissions by default. Especially since these specific roles aren't namespaced and are in fact cluster wide

Reply:
The Scheduler (coming soon) just needs to create/delete. But the operator on itself should need all these privileges so that CRs can be fully modified (manually for example) if needed and operations on it shouldn't be limited.